app/web_conn.go

```diff
@@ -104,8 +105,17 @@ func (c *WebConn) WritePump() {
 				return
 			}
 
+			var msgBytes []byte
```

Comment:
Yeah I wasn't sure if that was any better than passing it down as value but I can do that instead